### ISSUE TITLE
NPE in TCNetworkReceiver

### ISSUE THREAD

**Issue by tao1** [TagCommander/Android#14]
I got a crash on a OnePlus 8 (Android 11). I am using TagCommander core 4.6.4

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.net.NetworkInfo.getTypeName()' on a null object reference
       at com.tagcommander.lib.core.TCNetworkReceiver$1.onLost(TCNetworkReceiver.java:113)
       at android.net.ConnectivityManager$CallbackHandler.handleMessage(ConnectivityManager.java:3713)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:233)
       at android.os.HandlerThread.run(HandlerThread.java:67)
```

**Comment by CommandersAct** [comment]
Hello, 

Thank you for reaching out. we'll be investigating the issue asap for the upcoming releases.

**Comment by Aure77** [comment]
I also having crash on Android 9 & 10 devices. This issue has been fixed in 4.6.5+ ?

I see this in changelog:

> ~ fix for network receiver's crash on Android 11 and re-registering after a disable SDK in the same session.

But that relate to Android 11 only...
Can someone confirm ?

**Comment by CommandersAct** [comment]
Dear @Aure77, 

Thank you for reaching out, we confirm that the bug was fixed for all Android versions. 

cheers,
